org.tug.texworks.yaml

```diff
@@ -0,0 +1,143 @@
+app-id: org.tug.texworks
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+command: texworks
+rename-desktop-file: texworks.desktop
+rename-appdata-file: texworks.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
No, this doesn't work for rename-appdata-file.